pkg/hubble/container/ring.go

```diff
@@ -24,6 +26,13 @@ import (
 	"github.com/cilium/cilium/pkg/lock"
 )
 
+var (
+	// ErrConcurrentWrite indicates that a value was overwritten
+	ErrConcurrentWrite = errors.New("concurrent write detected")
```

Comment:
Your commit message mentions `ErrConcurrentRead`. I think that both names are misleading anyway as reading concurrently should not be a problem and a "concurrent write" error while reading is rather something unexpected. What about a more generic name like `ErrInvalidRead` for instance?

Reply:
Oh, the `ConcurrentRead` a typo in the commit messages. I'll think about the naming some more. I don't want it to be too generic, since the error is also presented to the user, and `invalid read` sounds like something is broken internally, when in fact they just tried to read stale data.